### PR TITLE
Scripts/Spells: Implemented Priest talent Blaze of Light

### PR DESCRIPTION
**Pull request by cristianvnt**
**Issues addressed:**

[Blaze of Light](https://www.wowhead.com/spell=215768/blaze-of-light) doesn't proc at all, and it should from Penance.

**Tests performed:**

tested ingame

### CODE REVIEW

File: src/server/scripts/Spells/spell_priest.cpp [hunk 1]
```diff
@@ -619,6 +622,30 @@ class spell_pri_benediction : public SpellScript
...
+    {
+        Unit* caster = eventInfo.GetActor();
+        Unit* target = eventInfo.GetProcTarget();
+        if (!caster || !target)
+            return;
+
+        if (caster->IsValidAttackTarget(target))
+        {
+            caster->CastSpell(target, SPELL_PRIEST_BLAZE_OF_LIGHT_DECREASE, aurEff);
+            return;
```
Comment (Nyr97) [inline src/server/scripts/Spells/spell_priest.cpp:638]: is there any reason you're returning here?

Comment (Mematoru23) [reply src/server/scripts/Spells/spell_priest.cpp:638]: theres no point since i check the target, right?

Comment (mdX7) [reply src/server/scripts/Spells/spell_priest.cpp:638]:
```c
if (caster->IsValidAttackTarget(target))
    caster->CastSpell(target, SPELL_PRIEST_BLAZE_OF_LIGHT_DECREASE, aurEff);
else
    caster->CastSpell(target, SPELL_PRIEST_BLAZE_OF_LIGHT_INCREASE, aurEff);
```

Comment (Mematoru23) [reply src/server/scripts/Spells/spell_priest.cpp:638]: i see i see

**Pull request merged**